src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,271 @@
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>incoming counter.</value>
+        public long IncomingCounter
+        {
+            get
+            {
+                long incoming;
+                long outgoing;
+                int ret = Interop.Stc.Info.GetCounter(_infoHandle, out incoming, out outgoing);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get incoming counter from info, Error - " + (StcError)ret);
+                }
+                return incoming;
+            }
+        }
+
+        /// <summary>
+        /// A property to get outgoing counter from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>outgoing counter.</value>
+        public long OutgoingCounter
+        {
+            get
+            {
+                long incoming;
+                long outgoing;
+                int ret = Interop.Stc.Info.GetCounter(_infoHandle, out incoming, out outgoing);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get outgoing counter from info, Error - " + (StcError)ret);
+                }
+                return outgoing;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the roaming type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>IsRoaming.</value>
+        public bool IsRoaming
+        {
+            get
+            {
+                RoamingType roaming;
+                int ret = Interop.Stc.Info.GetRoaming(_infoHandle, out roaming);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Roaming type from info, Error - " + (StcError)ret);
+                }
+
+                return roaming == RoamingType.Enabled;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the network protocol type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Network Protocol type.</value>
+        public NetworkProtocol Protocol
+        {
+            get
+            {
+                NetworkProtocol protocol;
+                int ret = Interop.Stc.Info.GetProtocol(_infoHandle, out protocol);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Protocol type from info, Error - " + (StcError)ret);
+                }
+                return protocol;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the process state of application from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Monitored process state.</value>
+        public ProcessStateType ProcessState()
```

Comment:
How about use property, and How about rename to `ApplicationState`
because, "Background", "Foreground" is a state of Application, of course process can has state like "Background", "Forground" with cgroup, but it seem not.

Reply:
Done. I missed to make it a property.